test/uk/gov/hmrc/cgtpropertydisposals/connectors/SubscriptionConnectorImplSpec.scala

```diff
@@ -107,7 +121,7 @@ class SubscriptionConnectorImplSpec extends WordSpec with Matchers with MockFact
       }
 
       "return an error when the future fails" in {
-        mockPost(s"http://host:123/subscribe", expectedHeaders, expectedRequest)(None)
+        mockPost(s"http://host:123/subscriptions/create/CGT", expectedHeaders, expectedRequest)(None)
 
```

Comment:
extract the DES URL into a `val` to avoid duplication

Reply:
also you are not doing string interpolation here so no need for the `s`

Reply:
I've factored out the url now into a `val`

Reply:
whoops, I did it on the wrong branch